<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sjzl</class>
 <widget class="QWidget" name="sjzl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>420</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>420</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>421</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;数据总览界面&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>19</y>
     <width>418</width>
     <height>371</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AcadEref</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>实时数据总览</string>
    </attribute>
    <widget class="QTableWidget" name="sstable">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>396</width>
       <height>301</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑 Light</family>
       <pointsize>6</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>165</x>
       <y>320</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>清空</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>320</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>开启实时总览</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>320</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>关闭实时总览</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>历史数据</string>
    </attribute>
    <widget class="QTableWidget" name="lstable">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>396</width>
       <height>201</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑 Light</family>
       <pointsize>6</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>396</width>
       <height>41</height>
      </rect>
     </property>
     <property name="title">
      <string>导入文件</string>
     </property>
     <widget class="QPushButton" name="choosebutton">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>12</y>
        <width>55</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑 Light</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>浏览</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lsline">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>12</y>
        <width>321</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑 Light</family>
       </font>
      </property>
     </widget>
     <zorder>lsline</zorder>
     <zorder>choosebutton</zorder>
    </widget>
    <widget class="QTabWidget" name="tabWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>243</y>
       <width>396</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑 Light</family>
       <pointsize>7</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>速度</string>
      </attribute>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>31</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>时间：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="vline">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>10</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="vcxbutton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>91</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>查询</string>
       </property>
      </widget>
      <widget class="QTableWidget" name="tableWidget">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>7</y>
         <width>275</width>
         <height>65</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>分贝</string>
      </attribute>
      <widget class="QTableWidget" name="tableWidget_2">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>7</y>
         <width>275</width>
         <height>65</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="fbline">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>10</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>31</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>时间：</string>
       </property>
      </widget>
      <widget class="QPushButton" name="fbcxbutton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>91</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>查询</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>酒精</string>
      </attribute>
      <widget class="QTableWidget" name="tableWidget_3">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>7</y>
         <width>275</width>
         <height>65</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="jjline">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>10</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>31</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>时间：</string>
       </property>
      </widget>
      <widget class="QPushButton" name="jjcxbutton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>91</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>查询</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_6">
      <attribute name="title">
       <string>温度</string>
      </attribute>
      <widget class="QTableWidget" name="tableWidget_4">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>7</y>
         <width>275</width>
         <height>65</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="wdline">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>10</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>31</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>时间：</string>
       </property>
      </widget>
      <widget class="QPushButton" name="wdcxbutton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>91</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>查询</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_7">
      <attribute name="title">
       <string>湿度</string>
      </attribute>
      <widget class="QTableWidget" name="tableWidget_5">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>7</y>
         <width>275</width>
         <height>65</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="sdline">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>10</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>31</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>时间：</string>
       </property>
      </widget>
      <widget class="QPushButton" name="sdcxbutton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>91</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>查询</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_8">
      <attribute name="title">
       <string>烟雾</string>
      </attribute>
      <widget class="QTableWidget" name="tableWidget_6">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>7</y>
         <width>275</width>
         <height>65</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="ywline">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>10</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>31</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>时间：</string>
       </property>
      </widget>
      <widget class="QPushButton" name="ywcxbutton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>91</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>查询</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_9">
      <attribute name="title">
       <string>胎压</string>
      </attribute>
      <widget class="QTableWidget" name="tableWidget_7">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>7</y>
         <width>275</width>
         <height>65</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="tyline">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>10</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>31</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>时间：</string>
       </property>
      </widget>
      <widget class="QPushButton" name="tycxbutton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>91</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>查询</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_10">
      <attribute name="title">
       <string>后车距</string>
      </attribute>
      <widget class="QTableWidget" name="tableWidget_8">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>7</y>
         <width>275</width>
         <height>65</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="dline">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>10</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>31</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>时间：</string>
       </property>
      </widget>
      <widget class="QPushButton" name="hcjcxbutton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>91</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>查询</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_11">
      <attribute name="title">
       <string>油量</string>
      </attribute>
      <widget class="QTableWidget" name="tableWidget_9">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>7</y>
         <width>275</width>
         <height>65</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="ylline">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>10</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>31</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>时间：</string>
       </property>
      </widget>
      <widget class="QPushButton" name="ylcxbutton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>91</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>查询</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_12">
    <attribute name="title">
     <string>数据可视化</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>396</width>
       <height>41</height>
      </rect>
     </property>
     <property name="title">
      <string>导入文件</string>
     </property>
     <widget class="QPushButton" name="choosebutton_2">
      <property name="geometry">
       <rect>
        <x>174</x>
        <y>12</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑 Light</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>浏览</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ksline">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>12</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑 Light</family>
        <pointsize>8</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLineEdit" name="sensorline">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>12</y>
        <width>125</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑 Light</family>
        <pointsize>8</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>10</y>
        <width>54</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>传感器：</string>
      </property>
     </widget>
     <zorder>ksline</zorder>
     <zorder>choosebutton_2</zorder>
     <zorder>sensorline</zorder>
     <zorder>label_11</zorder>
    </widget>
    <widget class="QChartView" name="chartView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>396</width>
       <height>291</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header>qtcharts//qchartview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
